### PR TITLE
SWC-7127, SWC-7128

### PR DESCRIPTION
**Pull request by jay-hodgson**

### CODE REVIEW

File: apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx [hunk 1]
```diff
@@ -108,6 +110,8 @@ export const SignUpdatedTermsOfUsePage = (
                 onClick={() => {
                   sessionStorage.setItem('skippedSigningToS', 'true')
                   restoreLastPlace(history)
+                  //if we did not redirect to a page in this app, then look for the redirect cookie
+                  processRedirectURLInOneSage()
```
Comment (jay-hodgson) [inline apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx:114]: SWC-7127: Fix the redirect issue after skipping or agreeing to the new ToS

File: packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts [hunk 2]
```diff
@@ -14,14 +14,19 @@ export function getOneSageBaseUrl(
   currentBackendEndpoint: string,
 ) {
   // If we're on synapse.org, then go to the accounts site that matches the current backend stack.
+  //backend endpoint may contain a trailing slash, so let's remove it if present:
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts:17]: SWC-7128: Fix the mapping issue

File: packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts [hunk 3]
```diff
@@ -31,11 +36,11 @@ export function getOneSageBaseUrl(
   }
 
   if (currentHostname === 'localhost' || currentHostname === '127.0.0.1') {
-    return `http://${currentHostname}:3000/`
+    return `http://${currentHostname}:3000`
```
Comment (jay-hodgson) [inline packages/synapse-react-client/src/utils/hooks/useOneSageURL.ts:39]: Just to make consistent with the other URLs that are returned (ONE_SAGE_STAGING_URL and ONE_SAGE_PRODUCTION_URL found in SynapseConstants, which do not have a trailing forward slash)

**Comment by jay-hodgson** [comment]
Merging now due to the tight deadline, but would like this to be reviewed

File: apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx (hunk 1 above)
Comment (nickgros) [reply apps/SageAccountWeb/src/components/SignUpdatedTermsOfUsePage.tsx:114]: One minor change suggested in #1314

**Pull request merged**